README.md

```diff
@@ -12,6 +13,20 @@ An R package to handle data from the clinical data management system (CDMS) [sec
 devtools::install_github("SwissClinicalTrialOrganisation/secuTrialR")
 ```
 
+## Recommended export options
+
+While the package strives to allow loading of as many types of secuTrial data exports
+as possible, there are certain export options which are less likely to cause issues.
+If possible it is suggested to export data which adheres to a suggested option set.
+Thus, we suggest to work with exports which:
+- are zipped
```

Comment:
If I'm not mistaken, we do not write explicitly inside of the readme that the CSV export format is required. Shall we add this here?

Reply:
yes, good idea.